scripts/deploy_hetzner.sh

```
@@ -0,0 +1,125 @@
+  apt-get install -y ca-certificates curl gnupg git ufw
+  install -m 0755 -d /etc/apt/keyrings
+  curl -fsSL "https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg" | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+  chmod a+r /etc/apt/keyrings/docker.gpg
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(. /etc/os-release; echo "$VERSION_CODENAME") stable" > /etc/apt/sources.list.d/docker.list
+  apt-get update -y
+  apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+  systemctl enable --now docker
+}
+
+clone_repo() {
+  if [[ -d $APP_DIR/.git ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)